src/main/java/org/mastodon/app/MastodonView.java

```diff
@@ -67,14 +67,7 @@
  *
  * @author Tobias Pietzsch
  */
-public class MastodonView<
-		M extends MastodonAppModel< ?, MV, ME >,
-		VG extends ViewGraph< MV, ME, V, E >,
-		MV extends AbstractSpot< MV, ME, ?, ?, ? >,
-		ME extends AbstractListenableEdge< ME, MV, ?, ? >,
-		V extends Vertex< E >,
-		E extends Edge< V > >
-	implements IMastodonView
+public class MastodonView< M extends MastodonAppModel< ?, MV, ME >, VG extends ViewGraph< MV, ME, V, E >, MV extends AbstractSpot< MV, ME, ?, ?, ? >, ME extends AbstractListenableEdge< ME, MV, ?, ? >, V extends Vertex< E >, E extends Edge< V > > implements IMastodonView
```

Comment:
This line looks way to long. I don't like that manual line breaks are automatically removed.